### PR TITLE
Subscriptions renderer

### PR DESCRIPTION
**Pull request by dvoytenko**
The intent with the `Render` class is to eventually provide the method `render(state|entitlements|???)`. This method will be called once entitlements have been resolved and it will translate the state/entitlements and call `setGrantState_`, `setLoginState_`, etc internally.

### CODE REVIEW

File: extensions/amp-subscriptions/0.1/amp-subscriptions.js [hunk 1]
```diff
@@ -82,33 +87,33 @@ export class SubscriptionService {
         .then(() => this.processEntitlement_());
   }
 
-  /**
-   * @private
-   */
+  /** @private */
   processEntitlement_() {
     // TODO(@prateekbh): process and unblock marup here.
+    this.renderer_.toggleLoading(false);
```
Comment (prateekbh) [inline extensions/amp-subscriptions/0.1/amp-subscriptions.js]: lets add a small test for this

File: extensions/amp-subscriptions/0.1/renderer.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  /**
+   * @param {string} type
+   * @param {?boolean} state
+   * @private
+   */
+  setState_(type, state) {
+    this.vsync_.mutate(() => {
+      this.getRootElement_().classList.toggle(
+          `${CSS_PREFIX}-${type}-unk`,
```
Comment (prateekbh) [inline extensions/amp-subscriptions/0.1/renderer.js]: is `unk` something we regularly use across AMP?
can we make it `unknown/none`?

Comment (dvoytenko) [reply extensions/amp-subscriptions/0.1/renderer.js]: This is a private class - not user visible. So, I'm ok keeping it slightly obscure. But if you feel strongly about it we expand.

File: extensions/amp-subscriptions/0.1/amp-subscriptions.js (hunk 1 above)
Comment (dvoytenko) [reply extensions/amp-subscriptions/0.1/amp-subscriptions.js]: I'm actually going to remove this for now and leave it for you to add. I don't yet have a clear "phases" picture with the integration code.

Comment (dvoytenko) [reply extensions/amp-subscriptions/0.1/amp-subscriptions.js]: Dropped.

**Pull request merged**